packages/core/src/components/map-controls/map-controls.template.ts

```diff
@@ -0,0 +1,48 @@
+import { checkFix, MapControls } from '@dso-toolkit/sources';
+import { html } from 'lit-html';
+
+import { SelectedBaseLayerChangeEvent } from '../map-base-layers/map-base-layers.interfaces';
+import { CheckedOverlaysChangeEvent, Overlay } from '../map-overlays/map-overlays.interfaces';
+
+export function mapControlsTemplate({
+  zoomIn,
+  zoomOut,
+  open,
+  baseLayers,
+  selectedBaseLayer,
+  baseLayerChange,
+  overlays,
+  checkedOverlays,
+  checkedOverlaysChange,
+  disableZoom
+}: MapControls) {
+  return html`
+    <div class="dsosb-map-container-mock" style="background-color: #efefef; height: 500px; position: relative; overflow: hidden">
```

Comment:
Ff uitzoeken wat mogelijk is om deze root node te verplaatsen naar `addParameter()` in `@dso-toolkit/sources`